platform/android/export/export_plugin.cpp

```diff
@@ -2529,7 +2529,7 @@ bool EditorExportPlatformAndroid::has_valid_export_configuration(const Ref<Edito
 		// Check for the bin directory.
 		Ref<DirAccess> da = DirAccess::open(java_sdk_path.path_join("bin"), &errn);
 		if (errn != OK) {
-			err += TTR("Invalid Java SDK path in Editor Settings.");
+			err += TTR("Invalid Java SDK path in Editor Settings.") + " ";
```

Comment:
Can you do a pass on the rest of the error strings in that method. From a quick look, I spotted a few others that were missing the space between the period and the next sentence.

Reply:
done
![image](https://github.com/user-attachments/assets/a0f5fa79-dd5e-48ef-a1a8-876fa245a323)